samples/contrib/versioned-pipeline-ci-samples/kaggle-ci-sample/create_pipeline_version_and_run.py

```diff
@@ -0,0 +1,54 @@
+import kfp
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--version_name', help='Required. Name of the new version. Must be unique.', type=str)
+parser.add_argument('--package_url', help='Required. pipeline package url', type=str)
+parser.add_argument('--pipeline_id', help = 'Required. pipeline id',type=str)
```

Comment:
nit: consistent style.
```suggestion
parser.add_argument('--pipeline_id', help='Required. pipeline id', type=str)
```

Reply:
thanks